src/Tag/Base/Tag.php

```diff
@@ -81,27 +82,41 @@ final public function id(?string $id): static
     /**
      * Add one or more CSS classes to the tag.
      *
-     * @param string|null ...$class One or many CSS classes.
+     * @param BackedEnum|string|null ...$class One or many CSS classes.
      */
-    final public function addClass(?string ...$class): static
+    final public function addClass(BackedEnum|string|null ...$class): static
     {
         $new = clone $this;
-        Html::addCssClass(
-            $new->attributes,
-            array_filter($class, static fn ($c) => $c !== null),
-        );
+        foreach ($class as $c) {
+            if ($c !== null) {
+                if ($c instanceof BackedEnum) {
+                    $c = is_string($c->value) ? $c->value : null;
+                }
+                Html::addCssClass($new->attributes, $c);
+            }
+        }
```

Comment:
Add backed enumeration support to `Html::addCssClass` method instead it.

Reply:
done.